### DISCUSSION TITLE
Dynamically change pages prop in Parallax. Page length changes based on content.

### DISCUSSION THREAD

**Discussion by fullspectrumprocessing** [pmndrs/react-spring#766] [Support]
## 🤓 Question
Hello Good Morning. Just wanted to know if anyone had a way to change the length of the parallax based on its content. Right now it seems that the length of the parallax is solely based on the 'pages' prop you pass when implementing it. So, when i put several components into the Parallax, some that do  scroll and some that are fixed, they get all jumbled up until I manually change the pages number. What do you think is the best way to dynamically change the parallax pages number for it to grow or shrink based on the content

**Comment by Taally** [comment]
No updates yet? It's a very important issue

**Reply by aleclarson** [reply]
The `Parallax` component isn't being actively developed.

Outside contributors will be needed.
